mcp_zammad/server.py

```
@@ -43,6 +49,293 @@
+                        obj[key].pop()  # remove from end
+
+            # Add metadata about truncation
+            meta = obj.setdefault("_meta", {})
+            meta.update({
+                "truncated": True,
+                "original_size": len(content),
+                "limit": limit,
+                "note": "Response truncated; reduce page/per_page or add filters."
+            })
+            return json.dumps(obj, indent=2, default=str)
+        except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)